core/src/include/ModelComponent.hpp

```diff
@@ -64,6 +64,8 @@ namespace Protected {
     inline constexpr TextTag WIND_V = "WIND_V"; // y(north)-ward component of wind, m s⁻¹
     inline constexpr TextTag ICE_U = "ICE_U"; // x(east)-ward ice velocity, m s⁻¹
     inline constexpr TextTag ICE_V = "ICE_V"; // y(north)-ward ice velocity, m s⁻¹
+    inline constexpr TextTag IO_STRESS_U = "IO_STRESS_U"; // x(east)-ward ice-ocean stress, Pa
+    inline constexpr TextTag IO_STRESS_V = "IO_STRESS_V"; // y(north)-ward ice-ocean stress, Pa
```

Comment:
I feel like x & y make more sense for stress components than u & v, but if u & v is the convention, then of course we should use that.

Reply:
Yes, x & y is better. Not sure why I did it like this, but there's no standard really. Using u and v for velocities and x and y for everything else is good, though.